packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_screen.dart

```diff
@@ -4,11 +4,21 @@
 
 import 'package:flutter/material.dart';
 
-import '../../../devtools_app.dart';
+import '../../shared/primitives/auto_dispose.dart';
+import '../../shared/primitives/utils.dart';
+import '../../shared/screen.dart';
+import '../../shared/table/table.dart';
+import '../../shared/table/table_data.dart';
+import '../../shared/theme.dart';
+import '../../shared/utils.dart';
 import 'deep_links_controller.dart';
 import 'deep_links_model.dart';
 
-const bundledDataRowHeight = 150.0;
+enum TableView {
```

Comment:
nit: maybe rename to `TableViewType` to make it more clear that this is a type enum and not a widget? When reading the code below,`TableView` sounded like a widget type